VDA-XP-Reward/README.md

```diff
@@ -0,0 +1,67 @@
+# Contract Explain
```

Comment:
`# Verida XP Rewards`

Reply:
This `README` file has separated into 2 parts:
- `Contract Explain`
- `Test & Deploy`

In the first part, it explains about the contract. In the second part, it explains how to test and deploy contract.

Should I rename the `Contract Explain` to `Verida XP Rewards`?

Reply:
> Should I rename the Contract Explain to Verida XP Rewards?

Yes pls

Reply:
Updated